.eslintrc

```diff
@@ -5,6 +5,6 @@
     "no-unused-vars": "error"
   },
   "parserOptions": {
-    "ecmaVersion": 6
+    "ecmaVersion": 2017
```

Comment:
This does not fix the problem mentioned before, it only allows `eslint` to pass. `async` and `await` are not available in Node `6` which this CLI still supports. `ecmaVersion` must remain at `6` for now.